schemas/logs/salesforce/login.yml

```diff
@@ -88,29 +88,29 @@ fields:
     required: false
     type: string
     description: >-
-        The identifier string returned by the browser used at login.
-        Example values are:
-            Go-http-client/1.1
-            Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv%3A50.0) Gecko/20100101 Firefox/50.0
-            Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36
+      The identifier string returned by the browser used at login.
+      Example values are:
+          Go-http-client/1.1
+          Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv%3A50.0) Gecko/20100101 Firefox/50.0
+          Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36
```

Comment:
I assume this is ok 🤷🏻‍♀️  it was changed by `make fmt`

Reply:
cc @alxarch

Reply:
Shouldn't be a problem @lindsey-w, I probably should have executed the auto-formatter command myself but I wasn't aware of it!

Reply:
Yes, the `prettier` command was added to `make fmt` _after_ this schema was added.